### PR TITLE
apm counter widget

### PR DESCRIPTION
**Pull request by TRONAS**

### CODE REVIEW

**Comment by TRONAS** [comment]
screen00021

File: LuaUI/Widgets/gui_apm_counter.lua [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+local FONT_SIZE = 15;
+local FONT_COLOR = {1, 1, 1, 0.8}
+--vars
+local actions=0;
+local playerId;
+local apm=0;
+local window;
+--loblob
+function widget:GameFrame()
+	_,_,_,actions,_ = GetPlayerStatistics(playerId);
```
Comment (sprunk) [inline LuaUI/Widgets/gui_apm_counter.lua:30]: the `_` are actually from the global namespace

File: LuaUI/Widgets/gui_apm_counter.lua [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+				fontSize = FONT_SIZE
+			}
+		}
+	};
+end
+function widget:PlayerChanged ()
+	SetSpectatingState();
+end
+
+function SetSpectatingState()
```
Comment (sprunk) [inline LuaUI/Widgets/gui_apm_counter.lua:62]: needlessly global

File: LuaUI/Widgets/gui_apm_counter.lua [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+local FONT_COLOR = {1, 1, 1, 0.8}
+--vars
+local actions=0;
+local playerId;
+local apm=0;
+local window;
+--loblob
+function widget:GameFrame()
+	_,_,_,actions,_ = GetPlayerStatistics(playerId);
+	apm = actions / (GetGameSeconds()/60)
```
Comment (sprunk) [inline LuaUI/Widgets/gui_apm_counter.lua:31]: trailing ws

**Comment by GoogleFrog** [comment]
What was wrong with this apart from a few suggested changes?

**Pull request closed without merging**